coveralls/reporter.py

```diff
@@ -69,19 +69,23 @@ def get_arcs(self, analysis):
             3. branch-number
             4. hits (we only get 1/0 from coverage.py)
         """
-        if not analysis.has_arcs():
+        try:
+            if not analysis.has_arcs():
+                return None
+            branch_lines = analysis.branch_lines()
+            executed = analysis.arcs_executed()
+            missing = analysis.arcs_missing()
+            branches = []
+            for l1, l2 in executed:
+                if l1 in branch_lines:
+                    branches.extend((l1, 0, abs(l2), 1))
+            for l1, l2 in missing:
+                if l1 in branch_lines:
+                    branches.extend((l1, 0, abs(l2), 0))
+            return branches
+        except KeyError:
```

Comment:
@goanpeca I'm debating whether we should log something here. `"Warning: branch coverage could not be collected. If you are using Python 3.6 or great, please ensure you have the latest version of coverage.py"`, or something like that. Thoughts?